apps/dotcom/client/index.html

```diff
@@ -1,34 +1,77 @@
 <!doctype html>
-<html lang="en">
+<html lang="en" dir="ltr">
 	<head>
 		<meta charset="UTF-8" />
 
 		<link rel="manifest" href="/manifest.webmanifest" />
+		<link rel="canonical" href="https://www.tldraw.com/" />
```

Comment:
can't use canonical here like this unfortunately :-/

the problem is that this `index.html` file is used on all paths for the site at the moment. so, effectively, on any room we'll be marking the canonical URL as the homepage. we'd have to add this in dynamically.

i tried doing this in https://github.com/tldraw/tldraw/pull/4471 but then realized it wasn't worth doing.